mmdet/datasets/transforms/transforms.py

```diff
@@ -2984,6 +2950,8 @@ def _transform_img(self, results: dict, patches: List[list]) -> None:
     def _transform_bboxes(self, results: dict, patches: List[list]) -> None:
         """Random erasing the bboxes."""
         bboxes = results['gt_bboxes']
+        assert isinstance(bboxes, HorizontalBoxes)
+        bboxes = bboxes.numpy()
```

Comment:
Add TODO here, indicating that we should use the operators of boxes in the future to simplify and unify the logic